src/Native/CMakeLists.txt

```diff
@@ -81,8 +81,8 @@ if (APPLE)
    
    # We cannot enable "stack-protector-strong" on OS X due to a bug in clang compiler (current version 7.0.2)  
    add_compile_options(-fstack-protector)  
-else ()  
-   add_compile_options(-fstack-protector-strong)  
+else ()
+   add_compile_options(-fstack-protector-strong)
```

Comment:
I am sorry. This change was unintentional. I had to remove else case for Ubuntu, as clang36 on my Ubuntu box was complaining, then I rebased and pushed. Later when i reverted it back, I thought the spaces are present  due to copy paste. Will revert it shortly.